README.md

```diff
@@ -1,8 +1,12 @@
 dojo
 ====
 
-**dojo** is the foundation package of version 1.x of the Dojo Toolkit. Sometimes 
-referred to as the “core”, it contains the most generally applicable sub-packages 
+This is the foundation package for the Dojo 1 Toolkit.  While still being maintained, new development is focused on Dojo 2.
+
+Checkout Dojo 2 [here](https://dojo.io/) or if you want a more detailed technical status and overview, checkout [`dojo/meta`](https://github.com/dojo/meta).
```

Comment:
Only change I would suggest is instead of "Dojo 2 [here](https://dojo.io/)", I would do "[Dojo 2](https://dojo.io/)"